Makefile

```diff
@@ -110,6 +110,14 @@ install-helm:
 
 .PHONY: e2e-local-bootstrap
 e2e-local-bootstrap:
-	kind create cluster --image kindest/node:v${KIND_K8S_VERSION}
+	kind create cluster --image kindest/node:v1.18.2@sha256:7b27a6d0f2517ff88ba444025beae41491b016bc6af573ba467b70c5e8e0d85f
```

Comment:
Do we need this change?

Reply:
Kind requires a very specific image for using Version 0.8 which is needed for networking between Kind Cluster and docker host